prometheus-metrics-core/src/test/java/io/prometheus/metrics/core/metrics/CounterWithCallbackTest.java

```diff
@@ -1,6 +1,42 @@
 package io.prometheus.metrics.core.metrics;
 
+import static org.junit.Assert.assertEquals;
+
+import io.prometheus.metrics.model.snapshots.CounterSnapshot;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import org.junit.Test;
+
 public class CounterWithCallbackTest {
 
-  // TODO :). Anyway, callbacks are implicitly covered by the JVM metrics tests as well.
+  @Test
+  public void testCounter() {
```

Comment:
wdyt of using assertj for new tests?

Reply:
As I'm not a maintainer on this project, I wasn't sure about adding new dependencies. I see you're doing the switch to assertj in https://github.com/prometheus/client_java/pull/1092